source/bsondump.txt

```diff
@@ -44,9 +44,9 @@ Versioning
 
 .. note:: Quick links to older documentation
 
-   - `MongoDB 4.2 bsondump <https://docs.mongodb.com/v4.2/reference/program/bsondump>`__
-   - `MongoDB 4.0 bsondump <https://docs.mongodb.com/v4.0/reference/program/bsondump>`__
-   - `MongoDB 3.6 bsondump <https://docs.mongodb.com/v3.6/reference/program/bsondump>`__
+   - `MongoDB 4.2 bsondump <https://www.mongodb.com/docs/v4.2/reference/program/bsondump>`__
+   - `MongoDB 4.0 bsondump <https://www.mongodb.com/docs/v4.0/reference/program/bsondump>`__
+   - `MongoDB 3.6 bsondump <https://www.mongodb.com/docs/v3.6/reference/program/bsondump>`__
```

Comment:
For these, rather than using full URLs, I think using the version roles may be better.

```suggestion
   - :v4.2:`MongoDB 4.2 bsondump </reference/program/bsondump>`
   - :v4.0:`MongoDB 4.0 bsondump </reference/program/bsondump>`
```

I think we can remove the v3.6 link because it's EOL.

This comment applies to the other db tools pages edited as part of this PR.